designs/vcenter-template-import.md

```diff
@@ -16,18 +16,27 @@ As an EKS Anywhere user:
 * I want to specify the template I want to import in a simple manner
 * I want the create command to know to use the pre-imported template
 
+### Users
+
+There are two types of users who would use this functionality. 
+
+* EKS-A maintainers who are in charge of creating the cluster
+* VCenter admins who only want to import the OVA beforehand to hand off to EKS-A maintainers
+
 
 ## Overview of Solution
 
 With this feature, a user can specify to import the latest OVA template before create by
-running a command. They would specify the cluster spec as input so that there is nothing 
-new that they need to know in terms of flags or different configuration. We would be introducing new
-subcommands as follows:
+running a command. They would either specify the cluster spec or the appropriate flags as input. 
+We would be introducing new subcommands as follows:
 
 ```
-eksctl anywhere import vsphere template -f <cluster_spec.yaml>
+eksctl anywhere import vsphere template -f <cluster_spec.yaml> --name <template_name>
+```
+or
+```
+eksctl anywhere import vsphere template --name <template_name> --path <vsphere_template_path> --os <os_family> --server <server_url> 
```

Comment:
nit: we might need a couple more flags here like datastore. We could just add `...` at the end to represent this.

Also, maybe we could simplify the interface by combining `--path` and `--name` into `--full-path`?